README.md

```diff
@@ -14,39 +14,53 @@ An overview of the architecture is shown below:
-| targetGrpcPort                  | 50051           | Port for target grpc server.                                                                                                                                                                           |
-| requestDelayMilliseconds        | 0               | Adds a delay between requests.                                                                                                                                                                         |
-
-### Warmup request format
+## Usage
+
+    mittens [flags]
+
+### Flags
+
+| Flag                               | Type    | Default value    | Description                                                                                                     |
+|:-----------------------------------|:--------|:-----------------|:----------------------------------------------------------------------------------------------------------------|
+| --concurrency                      | int     | 2                | Number of concurrent requests for warm up                                                                       |
+| --exit-after-warmup                | bool    | false            | If warm up process should finish after completion. This is useful to prevent container restarts.                |
+| --grpc-headers                     | strings | N/A              | gRPC headers to be sent with warm up requests.                                                                  |
+| --grpc-requests                    | strings | N/A              | gRPC request to be sent. Request is in '<service>/<method>[:message]' format. E.g. health/ping:{"key": "value"} |
+| --help, -h                         |         |                  | N/A Help for warmup-sidecar                                                                                     |
+| --http-headers                     | strings | N/A              | Http headers to be sent with warm up requests.                                                                  |
+| --http-requests                    | strings | N/A              | Http request to be sent. Request is in '<http-method>:<path>[:body]' format. E.g. post:/ping:{"key": "value"}   |
+| --probe-liveness-path              | strings | /alive           | Warm up sidecar liveness probe path                                                                             |
+| --probe-port                       | int     | /8000            | Warm up sidecar port for liveness and readiness probe                                                           |
+| --probe-readiness-path             | string  | /ready           | Warm up sidecar readiness probe path                                                                            |
+| --profile-cpu                      | string  | N/A              | Name of the file where to write CPU profile data                                                                |
```

Comment:
I believe it'd be better for mem/cpu profiling to be optional. 
Maybe the existing flags could be used for enabling/disabling and introduce 2 more flags for the filepath?